### ISSUE TITLE
No way to dismiss a tooltip on touch devices 

### ISSUE THREAD

**Issue by galenwilk** [HubSpot/tooltip#9]
On an iOS or other touch device, where hover events are triggered after a click, there does not appear to be way to get the tooltips to dismiss.

**Comment by muxcmux** [comment]
I had the same problem. Adding `openOn: 'hover click'` in the init options did the trick for me.

**Comment by cvrebert** [comment]
Based on https://developer.mozilla.org/en-US/docs/Web/Events/click#Safari_Mobile , another likely option is:
```js
if (is_iOS) {
  $(document.body).css({cursor: 'pointer'});
}
```

**Comment by aaronstezycki** [comment]
@muxcmux Can you elaborate on your fix? Is that and option you added within the manual initialisation?

i.e.:
``` 
new Tooltip({
  target: targetElement,
  position: 'top left',
  content: "My awesome <b>content</b>.",
  classes: 'my-tether-theme',
  openOn: 'hover click'
});
```

Also wondering if there's a fix for this in the works, because we're looking to use Tether, drop and tooltip in production.

**Comment by muxcmux** [comment]
@aaronstezycki Yes, that is correct and as far as I can remember correctly it was working fine on prod. Soon after that though, we abandoned the idea of tooltips on touch devices and just used inline text to aid users
